addon/-private/model.ts

```diff
@@ -402,6 +403,10 @@ export default class Model {
     return this._cache;
   }
 
+  get $store(): Store {
+    return this.$cache.store;
+  }
```

Comment:
I'd like to leave this getter off `Model` and force access through `modelInstance.$cache.store` to keep the relationship chain clear, and so that typings are clear that a cache's `store` may be undefined (vs. a model _must_ have a `$cache`).